Fenwick_Tree/cpp/fenwick.cpp

```diff
@@ -0,0 +1,45 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+#define MAXN 100010
+int fenwick[MAXN], n;
+
+void update(int index, int val) {
+  for(int i = index; i <= n; i += index & (-index)) 
+    fenwick[i] += val;
```

Comment:
Line contains following spacing inconsistencies:
- Spaces used instead of tabs.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpiz186cig/Fenwick_Tree/cpp/fenwick.cpp
+++ b/tmp/tmpiz186cig/Fenwick_Tree/cpp/fenwick.cpp
@@ -6,7 +6,7 @@
 
 void update(int index, int val) {
   for(int i = index; i <= n; i += index & (-index)) 
-    fenwick[i] += val;
+	fenwick[i] += val;
 }
 
 int getSum(int index) {
```